NOTICE.txt

```diff
@@ -31,3 +31,29 @@ bring it to our attention. Post an issue or email us:
 The attached notices are provided for information only.
 
 ------------------------------------------------------------------------------
+Copyright (c) 2012 - 2017, Lars Bilke
```

Comment:
Where is this line coming from?

I don't see it here: https://github.com/bilke/cmake-modules/blob/0503702f2ef733aa8f6a0c2015db89b085c5ac57/LICENSE_1_0.txt#L1

Reply:
this line is the mention. It comes directly from the file where the code is:
https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake#L1

Reply:
Gotcha. We already have that line in our copy of the `codecoverage.cmake` file. The notice.txt should just copy the license directly, and nothing else. Let's remove the copyright from our notice.txt file.

Reply:
see how Notice must be: 

Component. [Include the name of the component.]

Open Source License/Copyright Notice. [Include the text of each license found in that component’s source code, including copyright statements included in that license (e.g., Copyright © 2018 XYZ Corporation). The license is typically in a file called "LICENSE" or "COPYING."]

Additional Attribution (if any). [Usually found in a component’s "AUTHORS", "NOTICES", or "CONTRIBUTORS" (or some other similar variant) file.]

https://docs.opensource.microsoft.com/content/using/attribution-guidelines.html

Reply:
> including copyright statements included in that license

Right, exactly. We include it only if the license has it. The license that we are copying doesn't have a copyright statement in it, we don't need to add that by ourselves.

Reply:
```suggestion
```

Reply:
we do. basically we need to merge the specific file license (which usually adds the name mention) and the license from the license file, both mentions are expected in the notice.